.github/actions/aws-openshift-rosa-hcp-single-region-cleanup/action.yml

```diff
@@ -101,3 +101,10 @@ runs:
               if [ "${{ inputs.delete-ghost-clusters }}" == "true" ]; then
                 ${{ github.action_path }}/scripts/cleanup-ghost-clusters.sh ${{ inputs.max-age-hours-cluster }}
               fi
+
+        - name: Upload cleanup logs
```

Comment:
I'd suggest adding
`if: always()`

we do a check as part of the script that may exit 0 or 1 and would be nice to have the logs regardless.